cob_bringup/drivers/sick_s300.launch

```diff
@@ -3,18 +3,21 @@
 
 	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
-	<arg name="name" default="front"/>
+	<arg name="name"/>
```

Comment:
I thought the name argument should be mandatory rather than having a default value...